hedera-mirror-rest/api/v1/openapi.yml

```
@@ -1954,49 +1955,56 @@ components:
       schema:
         type: boolean
         default: true
+    contractIdPathParam:
+      name: contractId
+      description: The ID of the contract to return information for
+      example: 0.0.10 or 10
+      in: path
+      schema:
+        $ref: '#/components/schemas/EntityId'
     contractIdQueryParam:
-      name: contract.id
+      name: contractId
       description: The ID of the contract to return information for
       in: query
       examples:
         entityNumNoOperator:
```

```suggestion
        contractNumNoOperator:
```
applies to every `entityNum` in this examples section